### PR TITLE
Fixed over 80 issues. Left with 10.

### PR DESCRIPTION
**Pull request by bbborisk**
Hello! OK, so I fixed a bunch, but have no idea how to fix these 10. 

Screenshot from 2019-10-18 12-35-57

I don't know if it is acceptable, looking forward to feedback. This is an amazing opportunity for me to learn, so thank you very much!

cheers!

### CODE REVIEW

**Comment by bbborisk** [comment]
Hello! Thanks for the info. I created a new PR from another branch with
issue no. Hope this helps

**Comment by jywarren** [comment]
Oh wow, were these rubocop issues? And, could you drop a link to the original issue in here so we're all linked together? Thank you!!!

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def assets; end
   def promote_admin
     @user = User.find params[:id]
     unless @user.nil?
-      if logged_in_as(['admin'])
+      if logged_in_as(%w(admin))
```
Comment (jywarren) [inline app/controllers/admin_controller.rb]: This is in the rubocop config, but I'm not sure about it -- actually `['admin']` seems more readable and common of a usage... what do you think? Could we adapt the config to use this simpler format? Thank you!!!!

**Comment by bbborisk** [comment]
Sorry, I don't have my laptop with me for the weekend, will send it on
Monday if that's OK?

Cheers

**Comment by cesswairimu** [comment]
Yeah, its totally okay @bbborisk. Thanks

**Comment by bbborisk** [comment]
Ok, so here is the original issue link.
https://github.com/publiclab/plots2/issues/6340

Yup, there were around 94 issues. or something like that. I fixed some
syntax stuff and aligned the code, but these few I can't figure out on my
own. Thanks a lot for the help and support, this is my first time really
contributing and getting into programming profession after learning on my
own, so it means a LOT

cheers

**Comment by cesswairimu** [comment]
@bbborisk, I will give it a try and push to your branch if I succeed thanks so much for fixing these

**Comment by bbborisk** [comment]
Yeah, of course, thanks a bunch for working so hard :)

Cheers

File: app/controllers/wiki_controller.rb [hunk 2]
```diff
@@ -399,10 +399,10 @@ def methods
       @notes = Node.where(status: 1, type: %w(page))
         .where('node.nid IN (?)', nids)
         .where('(type = "note" OR type = "page" OR type = "map") AND node.status = 1 AND (node.title LIKE ? OR node_revisions.title LIKE ? OR node_revisions.body LIKE ? OR term_data.name = ?)',
-          '%' + params[:topic] + '%',
-          '%' + params[:topic] + '%',
-          '%' + params[:topic] + '%',
-          params[:topic])
+               '%' + params[:topic] + '%',
```
Comment (cesswairimu) [inline app/controllers/wiki_controller.rb]: Hey, I was not able to push to your branch. I was able to fix this by having them on one line and having the like start immediately below `'(type = "note"`..... like so
```
'(type = "n......
'%' + params[:topic] + '%', '%' + params[:topic] + '%', '%' + params[:topic] + '%', params[:topic])
``

Comment (bbborisk) [reply app/controllers/wiki_controller.rb]: Ok. Do I need to do something now?

**Comment by SidharthBansal** [comment]
Please rebase

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**